.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     - uses: actions/checkout@v1
     - uses: actions/setup-node@v1
       with:
-        node-version: '10.x'
+        node-version: '12.x'
```

Comment:
Sync Node.js version with version specified in the package.json file 

https://github.com/OctoLinker/api/blob/2758891e2afe15a931a44c22894a0d9c24dd0ba5/package.json#L38-L40